crates/rpc/src/server/mod.rs

```diff
@@ -30,7 +30,9 @@ pub async fn serve(config: RpcConfig) -> Result<(), ApiError> {
         .ok_or_else(|| ApiError::AddressResolutionFailed(config.endpoint.to_string()))?;
 
     Server::builder()
-        .add_service(rpc)
+        .accept_http1(true)
+        .add_service(rpc.clone())
```

Comment:
This should work because `rpc` has basically no state.

Reply:
Does this allow us to serve both HTTP/1.1 and HTTP/2.0 requests? That is, what would happen if we did just:
```Rust
Server::builder()
    .accept_http1(true)
    .add_service(tonic_web::enable(rpc))
    .serve(addr)
    .await
    .map_err(ApiError::ApiServeFailed)?;
```